lib/firebase_token_auth/client.rb

```diff
@@ -55,6 +55,10 @@ def user_search_by_email(email)
       admin_client.get_account_info({ email: [email] })&.users&.map(&:to_h)
     end
 
+    def signup_user(email, password, attribute_hash)
```

Comment:
attribute_hash is optional when signup. So I suppose to this is better.
```suggestion
    def signup_user(email, password, attribute_hash = {})
```